website/content/docs/agent/telemetry.mdx

```diff
@@ -519,47 +519,94 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.grpc.server.streams`                        | Measures the number of active gRPC streams handled by the server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | streams                           | gauge   |
 | `consul.xds.server.streams`                         | Measures the number of active xDS streams handled by the server split by protocol version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | streams                           | gauge   |
 
+
+## Server Workload
+
+** Requirements: **
+* Consul 1.12.0+
+
+We added new RPC metrics as of 1.12 to better understand A) the workload on a Consul server and B) where that workload is coming from. These metrics give insight into that:
+
+| Metric                                | Description                                               | Unit   | Type    |
+| ------------------------------------- | --------------------------------------------------------- | ------ | ------- |
+| `consul.rpc.server.call`              | Measures the elapsed time taken to complete an RPC call.  | ms     | gauge   |
```

Comment:
i am considering pointing out that some metrics may come out with value `==0` bc, in ms, it is zero, but no in smaller units 🤔